src/features/MoviesBrowser/ListPage/PersonTile/styled.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+
+export const StyledPersonTile = styled.div`
+    background-color: ${({ theme }) => theme.colors.tileBackground};
+    display: flex;
+    flex-direction: column;
+    align-items: center; 
+    padding: 16px;
+    width: 100%;
+
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.mobile}) {
+      grid-template-columns: 1fr;
+      align-items: center;
+      display: grid;
+      grid-gap: 16px;
+      width: 100%;
+      }
+`;
+
+export const StyledLink = styled(Link)`
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 22px;
+    line-height: 1.3;
+    color: ${({ theme }) => theme.colors.linkHeader};
+    text-decoration: none;
+    text-align: center;
+    margin: 8px 0px;
```

Comment:
To nie potrzebuje marginu.
```css
margin: none;
```

Reply:
No teraz już nie, bo poprawiony StyledLink, wcześniej potrzebował bo się zlewał z posterem :)